### PR TITLE
added cloudtrail, eks analysis and tests

### PR DESCRIPTION
**Pull request by GauravChawda**

### CODE REVIEW

File: cartography/intel/aws/__init__.py [hunk 1]
```diff
@@ -370,4 +370,10 @@ def start_aws_ingestion(neo4j_session: neo4j.Session, config: Config) -> None:
             neo4j_session,
             common_job_parameters,
         )
+
+        run_analysis_job(
+            'aws_cloudtrail_asset_exposure.json',
+            neo4j_session,
+            common_job_parameters,
+        )
```
Comment (ganeshjagdale9545) [inline cartography/intel/aws/__init__.py:378]: add for eks

File: cartography/intel/aws/cloudtrail.py [hunk 2]
```diff
@@ -51,6 +51,10 @@ def load_trails(neo4j_session: neo4j.Session, trails: Dict, current_aws_account_
         trail.has_custom_event_selectors = record.HasCustomEventSelectors,
         trail.has_insight_selectors = record.HasInsightSelectors
     WITH trail
+    MERGE (:S3Bucket{id: trail.s3bucket_name})<-[rel:HAS_BUCKET]-(trail)
```
Comment (ganeshjagdale9545) [inline cartography/intel/aws/cloudtrail.py:54]: create cleanup job for this

File: cartography/intel/aws/__init__.py (hunk 1 above)
Comment (GauravChawda) [reply cartography/intel/aws/__init__.py:378]: It was already there.
Line 368 to 372

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good

**Pull request merged**